client/tailscale/localclient.go

```diff
@@ -920,3 +911,24 @@ func tailscaledConnectHint() string {
 	}
 	return "not running?"
 }
+
+type jsonReader struct {
+	b   *bytes.Reader
+	err error // sticky JSON marshal error, if any
+}
+
+// jsonBody returns an io.Reader that marshals v as JSON and then reads its.
```

Comment:
s/its/it/

Reply:
I originally wrote "its JSON" and thought that was redundant.